tutorials/data-science-exploration/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Exploratory queries with Google BigQuery
+description: Use BigQuery to investigate arbitrarily large datasets.
+author: jerjou
+tags: Data Science, BigQuery
+date_published: 2017-05-10
+---
+[Google BigQuery](/bigquery) is a fully-managed service for data
+exploration using an SQL syntax. Once data has been loaded into the service, it
+can be queried at interactive speeds, without having to know beforehand what
+you're looking for.
+
```

Comment:
I guess that BigQuery is the only one listed in the _Big Data_ bucket on our products page, but as someone new to the space, I wouldn't intuitively know that BQ is better for this than Cloud SQL,  Cloud Bigtable, or Cloud Datastore. Are we assuming that our audience already knows the distinctions? Even if so, I don't think it would hurt to introduce the topic with a little context. Something along the lines of:

"Google Cloud Platform includes a variety of data services. The most useful of them for data science applications is BigQuery, because it x, y, and z. You can use BigQuery to use SQL syntax to query large datasets at interactive speeds..."